pages/events.js

```diff
@@ -30,7 +29,54 @@ export default function Events({ events }) {
       event.date.cfpClose ? new Date(event.date.cfpClose) > new Date() : false
     ),
   };
+  const filters = [
+    {
+      title: "Show all",
+      description: "List all events with no filters",
+      key: "all",
+      icon: FaListUl,
+      current: true,
+    },
+    {
+      title: "CFP open",
+      description: "You can submit a talk to this conference",
+      key: "cfpOpen",
+      icon: FaMicrophoneAlt,
+      current: false,
```

Comment:
Instead of this could we have a state for the current tab? the value could be the `key` .

Reply:
How would that work here?
![image](https://user-images.githubusercontent.com/95094057/211862771-fc511678-c7a5-4129-beb5-93edcb85756e.png)

Reply:
Since the `tabs` array of objects have `key` for each object. the state could be 

```jsx
const [eventType, setEventType] = useState("all");
```

initially. then the change function could fetch the `key` of the event type that the user click then set it to that.


the ui could be changed according to that